### PR TITLE
issue-181 add task should close

### PR DESCRIPTION
**Pull request by ssteuteville**
## Description of the PR
We collapse the header for add_task when a new task is added. Created a util method to avoid duplicating code. Let me know if you want that or anything else changed. Also, I wasn't able to get the tests running on windows (I left a comment on the issue) and since this is a DOM change I didn't see any examples of how you'd want this tested.

Happy to add unit test for the DOM manipulation if you can provide me an example.. Or we could move the util method into a new module that would be shared between components to help with code coverage, but that feels like we're just hiding lack of coverage. Let me know what you want.

### Issues Related
#181 

------

Please, before doing the PR make sure that you are following the intructions described in [CONTRIBUTING.md](https://github.com/angrykoala/gaucho/blob/master/CONTRIBUTING.md) and you have completed the following checklist

**Checklist**
- [x] Your PR is done against the `dev` branch
- [x] Your origin branch is updated with the latest changes from `dev`
- [x] All the tests and jshint are passing (`npm run test`)
- [x] Changelog is updated with a brief description of your changes (if required)
- [?] Your changes have tests

Thanks for contributing to Gaucho

### CODE REVIEW

**Comment by ssteuteville** [comment]
ok got the tests running locally with this command:
`istanbul cover node_modules/mocha/bin/_mocha -- -R spec` 

But I also noticed you have CI to run the tests as well.

File: app/common/utils.js [hunk 1]
```diff
@@ -27,4 +29,11 @@ module.exports = {
     isDevEnv() {
         return process.env.NODE_ENV === "dev";
     },
+    collapseHeader($el) {
```
Comment (angrykoala) [inline app/common/utils.js]: I like the idea of creating this function, but wouldn't it make more sense having this in materialize (or maybe elsewhere)?

keep in mind that utils.js (and everything under common) is intended to be used (or at least, to not depend) on either renderer or main proceses, while `collapseHeader` is exclusively a renderer process

**Comment by angrykoala** [comment]
I checked that this PR solves #181 correctly, thanks for contributing @ssteuteville 
I commented about a possible improvement in the code, please, check the comment before I merge to discuss possible changes :+1:

**Comment by ssteuteville** [comment]
I was thinking the same thing after I submitted PR. Code is updated!

Cheers

**Pull request merged**